pkg/pipelines/dryrun.go

```diff
@@ -0,0 +1,62 @@
+package pipelines
```

Comment:
I'd be inclined to move this into a separate package.

Reply:
Moved the files to a new package